src/components/DisplayList.test.tsx

```diff
@@ -1,46 +1,60 @@
 import React from "react";
 import { render, screen } from "@testing-library/react";
 import DisplayList from "./DisplayList";
+import userEvent from "@testing-library/user-event";
 
+const mockOnClick = async () => {};
 test("Has an ul", () => {
-  const todolist = new Array<string>();
-  render(<DisplayList list={todolist} />);
+  const todolist = new Array<any>();
```

Comment:
Why the change from strict type checking to lose?

Reply:
I didn't want to but now the `todolist` is an array of JSON objects that has the `id` and `task` when before it was just an array of tasks as `strings`. Also the JSON objects come back as `Promises` from the `fetch` method so I wasn't sure which type to use.